src/main/java/org/opensearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -300,7 +300,7 @@ protected void saveAnUpdateConfigs(final Client client, final RestRequest reques
 									   final SecurityDynamicConfiguration<?> configuration, OnSucessActionListener<IndexResponse> actionListener) {
 		final IndexRequest ir = new IndexRequest(this.opendistroIndex);
 
-		//final String type = "_doc";
+		//final String type = "_doc"; 
```

Comment:
Is this needed?

Reply:
No; I think this was on accident. Will be fixed.